src/content/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-and-dt.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: New Relic Android mobile monitoring with distributed tracing
+metaDescription: This page describes how to set up Android distributed tracing.
+redirects:
+  - /docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-mobile-and-dt
+---
+New Relic Android mobile monitoring agents support distributed tracing. This lets you see how your mobile app activity connects to related services.
+
+Watch this short video (approx. 2:15 minutes) to learn how to:
+
+* Find mobile distributed tracing data in New Relic.
+* Filter by standard and custom attributes.
+
+<Video id="tkAxeE5cT2Q" type="youtube" />
+
+## Requirements [#requirements]
+
+To use distributed tracing, you need at least Android agent version 6.0.0. We recommend you use the most recent agent.
+
+<Callout variant="tip">
+Infinite Tracing is not yet available for mobile monitoring.
+</Callout>
+
+## How to set up distributed tracing [#setup]
+
+For mobile agents that support this feature, it’s enabled by default. 
+
+If you prefer to turn off distributed tracing, see [Android feature flag](/docs/mobile-monitoring/new-relic-mobile-android/api-guides/android-agent-configuration-feature-flags/#dt).
+
+## Find data [#find-data]
+
+Mobile spans appear in any [New Relic distributed tracing UI](/docs/understand-dependencies/distributed-tracing/ui-data/additional-distributed-tracing-features-new-relic-one/) where those spans are part of a trace. 
+
+Here are some tips for finding and querying data:
+
+* You can find end-user-originating traces in any [New Relic One distributed tracing UI](/docs/understand-dependencies/distributed-tracing/ui-data/additional-distributed-tracing-features-new-relic-one).
+* In the distributed tracing UI, end-user spans are indicated with the ![New Relic distributed tracing browser span icon](./images/mobile-icon.png "distributed-tracing-browser-span-icon.png") icon.
+* To see a span's attributes, select [a span in the UI](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data#span-details).
+* Spans are reported as [Span data](/attribute-dictionary/?event=Span), and can be [queried in New Relic](/docs/using-new-relic/data/understand-data/query-new-relic-data).
+* Query tips:
+  * Query by browser app name by setting `mobileApp.name` to the browser app name.
+  * Query for traces containing at least one mobile app span with `mobileApp.name is not null`.
+  * Query for traces containing at least one back-end app with `appName is not null`.
+  * Query for traces containing both mobile and back-end spans by combining the two previous conditions.
+
+## Troubleshooting [#troubleshooting]
+
+If you don't see end-user spans, or are having other distributed tracing issues, see [Troubleshooting](/docs/apm/distributed-tracing/troubleshooting/troubleshooting-missing-trace-data#browser-data).
```

Comment:
I don't understand why this link points to the browser monitoring info. Does that apply to mobile too? Also, looks like the anchor is now #browser not #browser-data

Reply:
@barbnewrelic, thanks for checking this over! I think that browser anchor in the URL was a mistake, so I removed it.